params/protocol_params.go

```diff
@@ -159,7 +159,6 @@ const (
 	AdditionalGasForNonGoldCurrencies uint64 = 3*ExpectedGasForCreditToTransactions + ExpectedGasForDebitFromTransactions + ExpectedGasToReadErc20Balance
```

Comment:
Can we increase the `MaxGasForDebitFromTransactions`, `MaxGasForCreditToTransactions, and `MaxGasToReadErc20Balance` all to 100k?